.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,29 @@
+
+# [JIRA-ID](add JIRA link): User Story / Defect / Task Title
+<!--For e.g [ODC-100](https://jira.coreos.com/browse/ODC-100): Title of user story / defect / task -->
+
+## Root cause
+<!--Briefly describe the root cause & analysis of the problem-->
+
+## Solution Description
+<!--Describe your code changes in detail and explain the solution-->
+
+- [ ] Is U/X approved?
+<!-- If designer review required, tag @openshift/team-devconsole-ux and add screenshots/gifs  -->
```

Comment:
I'd say if it approved, show a link to a conversation / doc / etc.

Reply:
Modified the section as "Screen shots / Gifs for design review". If designer review required, developers shall tag @openshift/team-devconsole-ux and add screenshots/gifs. Is it better to get approval on PR?

Reply:
I'd say it's useful but not required. We are already bouncing up against the UX wall when it comes to addressing stories. We shouldn't make the PRs that worked off of designs to wait on them in the same fashion.

We'll have to iterate over the design anyways, so I say we don't "wait" for it, but it is useful for us to see it.

Reply:
They want to be made aware of any UI changes. Gives them a chance to review.
```suggestion
<!-- If change affects UI in any way, tag @openshift/team-devconsole-ux and add screenshots/gifs  -->
```

Reply:
I don't disagree @christianvogt... absolutely should **always** tag them on UI changes - no argument here. But making it a requirement to wait? I don't think that's a good idea. Tag and let them know, if they get to it we catch what we can... but waiting will just ask for more slowdown

Reply:
Yes I agree with your statement above.

Reply:
Updated the statement as suggested